src/locale/locales/km/messages.po

```diff
@@ -0,0 +1,8616 @@
+
+#: src/screens/StarterPack/Wizard/index.tsx:506
+msgid "<0>{0}</0> is included in your starter pack"
+msgstr "<0>{0}</0> ត្រូវបានរួមបញ្ចូលនៅក្នុងកញ្ចប់ចាប់ផ្តើមរបស់អ្នក"
+
+#: src/components/WhoCanReply.tsx:274
+msgid "<0>{0}</0> members"
+msgstr "សមាជិក <0>{0}</0>"
+
+#: src/components/dms/DateDivider.tsx:69
+msgid "<0>{date}</0> at {time}"
+msgstr "<0>{date}</0> នៅ {time}"
+
+#: src/screens/Settings/NotificationSettings.tsx:71
+msgid "<0>Experimental:</0> When this preference is enabled, you'll only receive reply and quote notifications from users you follow. We'll continue to add more controls here over time."
+msgstr "<0>ការពិសោធន៍៖</0> នៅពេលដែលចំណូលចិត្តនេះត្រូវបានបើក អ្នកនឹងទទួលតែការជូនដំណឹងអំពីការឆ្លើយតប និងការដកស្រង់ពីអ្នកប្រើប្រាស់ដែលអ្នកតាមដានប៉ុណ្ណោះ។ យើងនឹងបន្តបន្ថែមការគ្រប់គ្រងបន្ថែមទៀតនៅទីនេះតាមពេលវេលា"
+
+#: src/screens/StarterPack/Wizard/index.tsx:463
+msgid "<0>You</0> and<1> </1><2>{0} </2>are included in your starter pack"
+msgstr "<0>អ្នក</0> និង<1> </1><2>{0}</2> ត្រូវបានរួមបញ្ចូលនៅក្នុងកញ្ចប់ចាប់ផ្តើមរបស់អ្នក"
+
+#: src/screens/Profile/Header/Handle.tsx:52
+msgid "⚠Invalid Handle"
+msgstr ""
+
+#: src/components/dialogs/MutedWords.tsx:193
+msgid "24 hours"
+msgstr "២៤ ម៉ោង"
+
+#: src/screens/Login/LoginForm.tsx:250
+msgid "2FA Confirmation"
+msgstr "ការបញ្ជាក់ 2FA"
+
+#: src/components/dialogs/MutedWords.tsx:232
+msgid "30 days"
+msgstr "៣០ ថ្ងៃ"
+
+#: src/components/dialogs/MutedWords.tsx:217
+msgid "7 days"
+msgstr "៧ ថ្ងៃ"
+
+#: src/Navigation.tsx:362
+#: src/screens/Settings/AboutSettings.tsx:24
+#: src/screens/Settings/Settings.tsx:207
+#: src/screens/Settings/Settings.tsx:210
+msgid "About"
+msgstr "អំពី"
+
+#: src/view/com/util/ViewHeader.tsx:89
+#: src/view/screens/Search/Search.tsx:885
+msgid "Access navigation links and settings"
+msgstr "ចូលប្រើតំណរុករក និងការកំណត់"
+
+#: src/view/com/home/HomeHeaderLayoutMobile.tsx:56
+msgid "Access profile and other navigation links"
+msgstr "ចូលប្រើកម្រងព័ត៌មាន និងតំណរុករកផ្សេងទៀត"
+
+#: src/screens/Settings/AccessibilitySettings.tsx:42
+#: src/screens/Settings/Settings.tsx:183
+#: src/screens/Settings/Settings.tsx:186
+msgid "Accessibility"
+msgstr "ភាពងាយស្រួល"
+
+#: src/view/screens/Settings/index.tsx:455
+#~ msgid "Accessibility settings"
+#~ msgstr "ការកំណត់ភាពងាយស្រួល"
+
+#: src/Navigation.tsx:322
+msgid "Accessibility Settings"
+msgstr "ការកំណត់ភាពងាយស្រួល"
+
+#: src/Navigation.tsx:338
+#: src/screens/Login/LoginForm.tsx:176
+#: src/screens/Settings/AccountSettings.tsx:41
+#: src/screens/Settings/Settings.tsx:145
+#: src/screens/Settings/Settings.tsx:148
+msgid "Account"
+msgstr "គណនី"
+
+#: src/view/com/profile/ProfileMenu.tsx:132
+#: src/view/com/util/forms/PostDropdownBtnMenuItems.tsx:360
+msgid "Account blocked"
+msgstr "គណនីត្រូវបានរារាំង"
+
+#: src/view/com/profile/ProfileMenu.tsx:145
+msgid "Account followed"
+msgstr "គណនីបានធ្វើតាម"
+
+#: src/view/com/profile/ProfileMenu.tsx:108
+msgid "Account muted"
+msgstr "គណនីត្រូវបានបិទ"
```

Comment:
Mute is different from Close and Block so I don't think the same translates as បិទ it's should be:
Mute = បិទសម្លេង 
Block = បញ្ឈប់

Reply:
So what would you suggest about `Account muted` in khmer?